pyiron_base/job/script.py

```diff
@@ -31,6 +31,68 @@ class ScriptJob(GenericJob):
         project (ProjectHDFio): ProjectHDFio instance which points to the HDF5 file the job is stored in
         job_name (str): name of the job, which has to be unique within the project
 
+    Example:
+        To submit a notebook as a ScriptJob job to the pyiron job management system,
+
+            1. Create the notebook to be submitted, for ex. 'example.ipynb', and save it -- Can contain any code like:
+                    ```
+                    from pyiron import Project
```

Comment:
Shouldn't we be importing fmor `pyiron.atomistics`?

Reply:
Absolutely!

Reply:
Is it not `pyiron_atomistics`?

Reply:
Right now, all three variants work (`from pyiron`, `from pyiron_atomistics`, and `from pyiron.atomistics` all result in the same `Project`). In the future we will deprecate `from_pyiron`. The question is now if we use the original pyiron package as main module and the atomistic as sub-module (`pyiron.atomistics`) or if we directly use the `pyiron_atomistics` repository (both will work in the future - `pyiron_atomistics` will be accessible via `pyiron.atomistics`.
Since I would like to promote the overarching `pyiron` framework with its several sub-modules (`atomistics`, `experimental`, `continuum`), I would go for `pyiron.atomistics`

Reply:
`pyiron.atomistics` doesn't seem to have `Project` at all, but `pyiron_atomistics` does... I'm a bit confused.

Reply:
You are right.... I raised an issue in `pyiron` since this should become the nomenclature to use. However, right now we have a name clash between the `pyiron_atomistics.atomistics` module and the `pyiron_atomistics` of which the first is bound to `pyiron.atomistics` for backwards compatability...

Reply:
Cool! I will follow that issue and update the docs as soon as it has been fixed.